docs/getting-started/using-isis-first-steps/introduction-to-isis.md

```diff
@@ -0,0 +1,373 @@
+
+-----
+
+Parameter arguments are always made up of keyword value pairs. The
+keyword and value are separated by an equals sign just like reserved
+arguments, but they begin with an alphabetic letter instead of a dash.
+Parameters control what input data to use, what settings to use to
+process that input data, and where to put the output data. There are six
+different types of parameters:
+
+  - **Cube file names** - A cube file name is the location on disk of an
+    ISIS cube. It can be either the input to this program or where to
+    place the output. Input cubes, must be the output from some other
+    ISIS program. Most programs can process cubes from ISIS2, but some
+    information may be interpreted differently. Output cubes, will
+    always be the location to place processed results from the
+    application. When entering the filename of a cube it is not
+    necessary to enter the .cub extension. It will be added for you if
+    you do not type it.
+
+    ```
+    from=r0700563_lev1.cub
+    to= f431a62
+    ```
+
+  - **Data file names** - A data file name is the location on disk of a file
+    which is not an ISIS cube. This includes input files from missions
+    such as Mars Global Surveyor or Viking, and output files such as
+    that produced by the program **stats** or **isis2std** .
+
+    ```
+    to=myStatistic.dat
+    ```
+
+  - **Floating point numbers** - Floating point numbers are numeric values
+    with a whole part and/or a fractional part. Parameters with this
+    type can be given values like 1.0, 0.7823, 127 or 0.31416E+1
+
+    ```
+    radius=317.681
+    ```
+
+  - **Integers** - Integer numbers have only a whole part. Parameters of
+    this type can be given values like 0, 2, -18223 or 255.
+
+    ```
+    lines=1024
+    ```
+
+  - **Booleans** - Boolean parameters are used as flags to indicate true or
+    false. The ISIS user interface allows true values to be represented
+    by True, T, Yes or Y. False values can be False, F, No or N. Any
+    mixture of upper and lower case letters is allowed. The name of the
+    parameter for boolean's is usually a question such as USEDEM. This
+    should be read as "Do you want the program to use a DEM when it
+    processes the input?"
+
+    ```
+    CREATESPECIAL=FALSE
+    emission=true
+    ```
+
+  - **Strings** - A string is a simple sequence of printable characters. For
+    example "MARS", "BiLinear" or "The quick brown fox."
+
+    ```
+    bittype=real
+    ```
+
+## Application Running Modes
+
+-----
+
+All non-interactive programs can run in one of two modes, text mode
+(runs without launching a window) or graphical mode (launches a window
+for providing parameters). Which mode the program runs in depends on
+what arguments you supplied on the command line. If any parameter
+arguments are present on the command line, the program will run in text
+mode unless explicitly overridden using the reserved argument `-gui`.
```

Comment:
There are a couple instances of an argument being mentioned as plain text, instead of the backticked code syntax like here. Perhaps it's preference but I think the mention of arguments (so far I've seen, `-gui`, `-help`, `-restore`) in code form help differentiate and break up the text.

Reply:
Good catch, I agree it helps break up the text and clarify the flags.  I code-quoted these and a few more things I found in the intro-to-isis doc.